mapbox_gl_web/lib/src/mapbox_web_gl_platform.dart

```diff
@@ -695,6 +695,12 @@ class MapboxWebGlPlatform extends MapboxGlPlatform
     _map.setFilter(layerId, filter);
   }
 
+  @override
+  Future<void> setVisibility(String layerId, bool isVisible) async {
+    _map.setLayoutProperty(
```

Comment:
Would this throw if the layer does not exist?
If so this would be the only platform were one gets in this situation right?